### ISSUE TITLE
WAVE reports empty link on homepage 

### ISSUE THREAD

**Issue by h-parekh** [samvera-labs/nurax-pre2023#222]
### Descriptive summary
Hyrax Version: 2.1.0.beta1 
Hyrax Branch: v2.1.0.beta1_qa 
Hyrax Revision: 8fe35f54e067fd499a4467b9ccbefa48233d43a2

When new items show up under 'Recently uploaded' tab of the homepage, WAVE accessibility tool reports an error for 'Empty Link'. This is non-compliance to section 508 as well as WCAG standards.

The 'a' tag in div.col-sm-2 does not have text. 
```html
<div id="recent_docs">
    <ol class="list-group list-group-striped recent_uploads">
      <li class="recent-item">
    <div class="row">
      <div class="col-sm-2">
        <a href="/concern/generic_works/zw12z528p?locale=en">
          </a><a data-context-href="/catalog/zw12z528p/track?locale=en" href="/concern/generic_works/zw12z528p?locale=en"><img suppress_link="true" src="/assets/work-ff055336041c3f7d310ad69109eda4a887b16ec501f35afc0a547c4adb97ee72.png" alt="Work"></a>
      </div>
      <div class="col-sm-10">
        <span class="sr-only">Title</span><h3><a href="/concern/generic_works/zw12z528p?locale=en">Generic_work</a></h3>
        <p>
          <span class="recent-label">Depositor:</span> <a href="/users/localhost_hparekh@nd-dot-edu">localhost_hparekh@nd.edu</a>
        </p>
        <p>
          <span class="recent-label">Keywords:</span> <a href="/catalog?f%5Bkeyword_sim%5D%5B%5D=Generic_work&amp;locale=en">Generic_work</a>
        </p>
      </div>
    </div>
</li>

    </ol>
  </div>
```

#### What It Means
A link contains no text.
#### Why It Matters
If a link contains no text, the function or purpose of the link will not be presented to the user. This can introduce confusion for keyboard and screen reader users.
#### How to Fix It
Remove the empty link or provide text within the link that describes the functionality and/or target of that link.

### Expected behavior
WAVE tool should not report this error

### Actual behavior
WAVE tool reports an error

### Steps to reproduce the behavior
1. Install the WAVE [extension for chrome](https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh)
2. On a vanilla hyrax instance, load the homepage in a browser
3. You can reproduce this on Nurax as well, but there will be other contrast errors that have been fixed in Vanilla Hyrax.
3. Click on the WAVE tool's icon and you will see the error on the 'Details' tab of WAVE tool
